### PR TITLE
fix(modal): change variable's name on modal to zindex

### PR DESCRIPTION
**Pull request by jowjow22**
## Infos

[Task](https://juntossomosmais.monday.com/boards/XXX/pulses/XXX)

#### What is being delivered?

Setting the new value for z-index on atom modal, the motivation is that ionic sets it too high and most of other UI libraries doesn't use this value (the value for it before was 20002) 

#### What impacts?

Atom modal component styling and Atom steps modal component

#### Reversal plan

Re release of the previous version and revert on main branch

#### Evidences

Media(images, gifs or videos) that shows the result of your work.

### CODE REVIEW

File: packages/core/src/components/modal/modal.scss [hunk 1]
```diff
@@ -1,12 +1,12 @@
 @import '~@atomium/scss-utils/index';
 
 :host {
-  --atom-modal-z-index: var(--zindex-1000);
+  --atom-modal-zindex: var(--zindex-1000);
```
Comment (felipefialho) [inline packages/core/src/components/modal/modal.scss:4]: we need to update docs

Comment (jowjow22) [reply packages/core/src/components/modal/modal.scss:4]: even if it's a local variable? i've followed the atom-tag example and it didn't had a example on docs :thinking:

Comment (felipefialho) [reply packages/core/src/components/modal/modal.scss:4]: [like a select example](https://juntossomosmais.github.io/atomium/?path=/docs/components-select--docs)

image

Comment (jowjow22) [reply packages/core/src/components/modal/modal.scss:4]: done it here!! [5efe8ab](https://github.com/juntossomosmais/atomium/pull/617/commits/5efe8abba05d32cdea8b157d550e546ffdf78bb1)

Comment (jowjow22) [reply packages/core/src/components/modal/modal.scss:4]: thanks for the example

File: packages/core/src/components/modal/stories/modal.args.ts [hunk 2]
```diff
@@ -160,6 +160,13 @@ export const ModalStoryArgs = {
         category: Category.SLOTS,
       },
     },
+    '--atom-modal-zindex': {
```
Comment (felipefialho) [inline packages/core/src/components/modal/stories/modal.args.ts:163]: we need to add it to docs of `atom-steps-modal` too

Comment (jowjow22) [reply packages/core/src/components/modal/stories/modal.args.ts:163]: done at [ff7cc0d](https://github.com/juntossomosmais/atomium/pull/617/commits/ff7cc0d21f35c136c5f8de648288ea63ff2e289a)

**Review by felipefialho — APPROVED** [review]

**Review by tassioFront — APPROVED** [review]

**Pull request merged**